docs/solutions/sql_server_always_on_availability_groups/configuration.tf

```diff
@@ -0,0 +1,112 @@
+// vcn_dns_name is the outermost prefix in the <vcn_dns_name>.oraclevcn.com domain.
+variable "vcn_dns_name" {
+  default = "SQLAlwaysOn"
+}
+
+// vcn_cidr_block defines the IP address pool for the network.
+// Check the locals configuration below for subnet declarations
+variable "vcn_cidr_block" {
+  default = "10.0.0.0/19"
+}
+
+// Compartment Name. This will only be used if the compartment_id variable in the terraform.tfvars file is not set.
+variable "compartment_name" {
+  default = "SQLAlwaysOn"
+}
+
+// Compartment Description. This will only be used if the compartment_id variable in the terraform.tfvars file is not set.
+variable "compartment_description" {
+  default = "SQL Always On compartment"
+}
+
+// How many availability domains are going to be used. Minimum 2, Maximum 3.
+variable "ad_count" {
+  default = "3"
+}
+
+variable "local_dns_server" {
+  default = "1.1.1.1"
+}
+
+// In which AD deploy the Witness servers
+variable "witness_deployment" {
+  default = "2"
+}
+
+// Windows 2012
+variable "image_id" {
```

Comment:
This will change with the region - see the recommended pattern at https://github.com/oracle/terraform-provider-oci/blob/master/docs/examples/compute/multi_vnic/multi_vnic.tf#L22